test/format/config.h

```diff
@@ -329,6 +329,9 @@ static CONFIG c[] = {
     "isolation level (random | read-uncommitted | read-committed | snapshot)", C_IGNORE | C_STRING,
     0, 0, 0, NULL, &g.c_isolation},
 
+  {"transaction.rollback_to_stable", "if rollback_to_stable should be called", C_BOOL, 0, 0, 0,
+    &g.c_txn_rollback_to_stable, NULL},
+
```

Comment:
There's no base percentage to configure this, was that deliberate? That is, this won't ever be set unless it's set on the command line?